From: Auto-Generated
Date: Mon, 17 Feb 2026 00:00:00 +0000
Subject: [PATCH] Add missing cmath header

Add the required <cmath> header to numeric_sensor.cpp to fix compilation
errors with std::isnan() calls.

---
 numeric_sensor.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/numeric_sensor.cpp b/numeric_sensor.cpp
index 1234567..abcdefg 100644
--- a/numeric_sensor.cpp
+++ b/numeric_sensor.cpp
@@ -18,6 +18,7 @@
 
 #include "numeric_sensor.hpp"
 
+#include <cmath>
 #include "threshold_helper.hpp"
 
 #include <phosphor-logging/log.hpp>
